Lesson22/duygu/carousel/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Carousel</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <div class="carousel">
+      <div class="carousel-container">
+        <img
+          class="active"
+          src="https://images.unsplash.com/photo-1682687982029-edb9aecf5f89?ixlib=rb-4.0.3&ixid=M3wxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2071&q=80"
+          alt="Image 1"
+        />
+        <img
+          src="https://images.unsplash.com/photo-1682695798522-6e208131916d?ixlib=rb-4.0.3&ixid=M3wxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2070&q=80"
+          alt="Image 2"
+        />
+        <img
+          src="https://images.unsplash.com/photo-1682687220063-4742bd7fd538?ixlib=rb-4.0.3&ixid=M3wxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1975&q=80"
+          alt="Image 3"
+        />
+      </div>
+      <div class="carousel-navigation">
+        <button class="carousel-btn prev-btn" id="prev-btn">Prev</button>
+        <button class="carousel-btn next-btn" id="next-btn">Next</button>
+      </div>
+      <div class="carousel-indicators">
+        <div class="indicator active"></div>
+        <div class="indicator"></div>
+        <div class="indicator"></div>
+      </div>
+      <div class="carousel-stostanavigation">
```

Comment:
It would be good to style stop and start buttons in a better way. E.g. you can do something like this:
```
      <div class="carousel-stostanavigation">
      <div class="carousel-stop">
        <button class="carousel-btn-transparent stop-btn" id="stop-btn">||</button>
      </div>
      <div class="carousel-start hidden">
        <button class="carousel-btn-transparent start-btn" id="start-btn">></button>
      </div>
      </div>
```

CSS:
```

.carousel-stostanavigation {
  position: absolute;  
  top: 50%;
  transform: translateY(-50%);
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  opacity: 1;
}

.hidden {
  display: none;
}
.carousel-btn-transparent {
  border-radius: 50%;
  /* width: 40px;
  height: 40px; */
  border: none;
  font-size: 36px;
  padding: 10px 20px;
  color: #242424;
  opacity: 0.5;
}
```

Then in JS, depending on the state of the slider, you can toggle class hidden for start and stop buttons.